system/jlib/jtrace.cpp

```diff
@@ -67,6 +67,117 @@ namespace opentel_trace = opentelemetry::trace;
 
 using namespace ln_uid;
 
+/**
```

Comment:
Could do with a comment to indicate it is based on code in the otel library, and why this is overwritten instead of using the default.

Reply:
See line 77-81